proto/docs/arbitration.md

```diff
@@ -1,74 +1,125 @@
-## Arbitration
```

Comment:
Hi Waqar can you explain what the expected behavior is when we have an active session with no role config and another session with a given non zero role_id? 

If I receive a stream when there is no role config  (i.e. role_id = 0) I assume this stream will handle all the P4 pipeline entities. Should we explicitly say if you have an active session with no role_id it means the master will be able to handle "all" entities? Or do we want to disallow not giving role_id? I suggest the first for backward compatibility.